src/tests/Microsoft.PowerFx.Connectors.Tests/PowerAppsForMakersTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.OpenApi.Models;
+using Microsoft.PowerFx.Functions;
+using Microsoft.PowerFx.Tests;
+using Microsoft.PowerFx.Types;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.PowerFx.Connectors.Tests
+{
+    public class PowerAppsForMakersTests : BaseConnectorTest
+    {
+        public PowerAppsForMakersTests(ITestOutputHelper output)
+            : base(output, @"Swagger\PowerAppsForMakers.json")
+        {
+        }
+
+        public override string GetNamespace() => "PowerAppsForMakers";
+
+        public override string GetEnvironment() => "a323f485-54aa-e556-937d-86727a0c5ac0";
+
+        public override string GetEndpoint() => "tip1002-002.azure-apihub.net";
+
+        public override string GetConnectionId() => "763ee25d4f4046debbfcddcf9175db97";        
+
+        [Fact]
+        public void PowerAppsForMakers_EnumFuncs()
+        {
+            EnumerateFunctions();
+        }
+
+        /*
+            EditAppRoleAssignment
+            EditConnectionRoleAssignment
+            EditConnectorRoleAssignment
+            GetApp
+            GetAppRoleAssignment
+            GetApps
+            GetAppVersions
+            GetConnectionRoleAssignment
+            GetConnections
+            GetConnector
+            GetConnectorRoleAssignment
+            GetConnectors
+            GetEnvironments
+            PublishApp
+            RemoveApp
+            RemoveConnection
+            RemoveConnector
+            RestoreAppVersion
+            SetAppDisplayName
+        */
+
+        // To run this test
+        // - get an working Aurora or working Power Apps environment
+        // - create a canva, add "PowerApps for Makers" connector and run expression below (GetConnectors)
+        // - set GetEnvironment(), GetEndpoint(), GetConnectionId(), GetJWTToken() values in this class
+        // - run the test
+        // NOTE: This test is very slow (takes ~15 minutes)
+        [Fact(Skip="Only for internal use")]
+        public async Task PowerAppsForMakers_GetConnectors()
```

Comment:
This is fine disabled, but eventually we should move this out to another repo.

Reply:
Will be moved in a later PR